lib/nostrum/struct/user.ex

```diff
@@ -153,6 +158,7 @@ defmodule Nostrum.Struct.User do
       map
       |> Map.new(fn {k, v} -> {Util.maybe_to_atom(k), v} end)
       |> Map.update(:id, nil, &Util.cast(&1, Snowflake))
+      |> Map.update(:public_flags, 0, &Flags.from_integer(&1))
```

Comment:
If we set this to 0 here as a default then the spec will be wrong. I think we should leave the integer as it is, and if users want to act on the flags then they can use your function to do so.

Reply:
Craig! Thank you for the review. Addressed in 3cdff15fc4f6658d239630b8d669ffdc458fdeb8